README.md

```diff
@@ -1,2 +1,2 @@
-# entity_reference_display_switcher
-Drupal 8 module provides formatter for entity reference fields, which allows to switch entity reference field view mode
+# Advanced Entity Reference Formatter
+Drupal 8 module that provides advanced formatter for entity reference fields, allowing to select different view modes per target bundle or delta item.
```

Comment:
"Advanced" sounds like a broad concept and according to description:
`allowing to select different view modes per target bundle or delta item` it is particulary focused on view mode selection so mb something like "View mode - Entity Reference Formatter" or "View display - Entity Reference Formatter" may be more accurate.

Reply:
It was renamed according to [this discussion](https://projects.skilld.cloud/skilld/ski-contrib/issues/74#note_266428) with @davidferlay

Reply:
@davidferlay what do you think about Juan's comment?

Reply:
the core thing of this module is to make "multiple" something that is usually "unique" > I believe this should be self-evident from module naming : 
I guess it should be something like `Entity Reference Multiple View Modes Formatter` but may be simpler)

Reply:
> sorlov
> Entity Reference Multiple View Modes Formatter too long i suppose
> 
> dferlay
> could be ok for human name, but machine name should be shorter for sure
> 
> sorlov
> maybe entity_reference_dynamic_display ?
> don't like word Multiple here
> as we don't show multiple view modes for reference
> 
> dferlay
> do we not ?)
> 
> sorlov
> we show 1 view mode for 1 referenced entity)
> from my opinion, multiple view modes -> when you show several view modes for 1 entity
> 
> dferlay
> same entity here for ex, but different bundles
> no?
> 
> sorlov
> you show rendered entity, but exact view_mode is selected based on some conditions
> 
> dferlay
> hm
> entity_reference_dynamic_display
> good for me then



TLDR: let's go with `Entity Reference Dynamic Display` (`entity_reference_dynamic_display`)

Reply:
Done